utils/s2n_random.c

```diff
@@ -325,24 +325,43 @@ int s2n_get_rdrand_data(struct s2n_blob *out)
     struct s2n_stuffer stuffer = {{0}};
     union {
         uint64_t u64;
+        struct {
+            /* since we check first that we're on intel, we can safely assume little endian. */
+            uint32_t uLow;
+            uint32_t uHigh;
+        } i386_fields;
         uint8_t u8[8];
     } output;
 
     GUARD(s2n_stuffer_init(&stuffer, out));
-
     while ((space_remaining = s2n_stuffer_space_remaining(&stuffer))) {
         int success = 0;
 
+        output.i386_fields.uLow = 0;
+        output.i386_fields.uHigh = 0;
```

Comment:
as above

Reply:
I just set u64 to 0, since that handles the whole thing.